src/shared/components/Types/TypesIcon.tsx

```diff
@@ -1,8 +1,19 @@
 import * as React from 'react';
-import { Avatar, Tooltip } from 'antd';
+import { Avatar, Tooltip, Tag } from 'antd';
 import * as Identicon from 'identicon.js';
 import './Types.less';
-import * as md5 from "md5"
+import * as md5 from 'md5';
+const { getPaletteFromURL } = require('color-thief-node');
```

Comment:
can you not do an `import { getPaletteFromURL } from 'color-thief-node` ?

Reply:
I think I was being lazy, if you use import then typescript will complain that this dep doesn't have a declaration file. I've added a type dec in our `custom.d.ts` file to fix that.

It's a wacky dependency but I think it's better than the original library because of node support, which I have plans for...